setup_tests.sh

```diff
@@ -23,7 +23,7 @@ if [[ ! -f "./web/test-data/micro_nor_cal.osm.pbf" ]]; then
     if [[ $OSTYPE == 'darwin'* ]]; then
       brew install osmfilter
     else
-      apt install -y osmctools
+      sudo apt install -y osmctools
```

Comment:
note: gh actions made me add this, basically. This command worked fine in circle but I think permissions are different in the actions setup